test/unit/utils/distance.utils.test.js

```diff
@@ -0,0 +1,169 @@
+import {
+  getSquaredEuclideanDistance,
+  isValidPosition,
+} from '#utils/distance.utils'
+
+describe('Distance utils', () => {
+  describe('getManhattanDistance', () => {
+    test.each([
```

Comment:
I find that the file has gotten very large because of the fixture here. Would you make any changes to improve the readability?